slices/systemd.yaml

```diff
@@ -0,0 +1,635 @@
+      /usr/lib/systemd/systemd-cryptsetup:
+      /usr/lib/systemd/systemd-executor:
+      /usr/lib/systemd/systemd-fsck:
+      /usr/lib/systemd/systemd-fsckd:
+      /usr/lib/systemd/systemd-growfs:
+      /usr/lib/systemd/systemd-hibernate-resume:
+      /usr/lib/systemd/systemd-hostnamed:
+      /usr/lib/systemd/systemd-initctl:
+      /usr/lib/systemd/systemd-integritysetup:
+      /usr/lib/systemd/systemd-journald:
+      /usr/lib/systemd/systemd-localed:
+      /usr/lib/systemd/systemd-logind:
+      /usr/lib/systemd/systemd-makefs:
+      /usr/lib/systemd/systemd-measure:
+        arch: [amd64, arm64, riscv64]
+      /usr/lib/systemd/systemd-modules-load:
+      /usr/lib/systemd/systemd-network-generator:
+      /usr/lib/systemd/systemd-networkd:
+      /usr/lib/systemd/systemd-networkd-wait-online:
+      /usr/lib/systemd/systemd-pcrextend:
+        arch: [amd64, arm64, riscv64]
+      /usr/lib/systemd/systemd-pcrlock:
+        arch: [amd64, arm64, riscv64]
+      /usr/lib/systemd/systemd-pstore:
+      /usr/lib/systemd/systemd-quotacheck:
+      /usr/lib/systemd/systemd-random-seed:
+      /usr/lib/systemd/systemd-remount-fs:
+      /usr/lib/systemd/systemd-reply-password:
+      /usr/lib/systemd/systemd-rfkill:
+      /usr/lib/systemd/systemd-shutdown:
+      /usr/lib/systemd/systemd-sleep:
+      /usr/lib/systemd/systemd-socket-proxyd:
+      /usr/lib/systemd/systemd-storagetm:
+      /usr/lib/systemd/systemd-sulogin-shell:
+      /usr/lib/systemd/systemd-sysctl:
+      /usr/lib/systemd/systemd-sysroot-fstab-check:
+      /usr/lib/systemd/systemd-sysupdate:
+      /usr/lib/systemd/systemd-sysv-install:
+      /usr/lib/systemd/systemd-time-wait-sync:
+      /usr/lib/systemd/systemd-timedated:
+      /usr/lib/systemd/systemd-tpm2-setup:
+        arch: [amd64, arm64, riscv64]
+      /usr/lib/systemd/systemd-update-done:
+      /usr/lib/systemd/systemd-update-utmp:
+      /usr/lib/systemd/systemd-user-runtime-dir:
+      /usr/lib/systemd/systemd-user-sessions:
+      /usr/lib/systemd/systemd-veritysetup:
+      /usr/lib/systemd/systemd-volatile-root:
+      /usr/lib/systemd/systemd-xdg-autostart-condition:
+
+  config:
+    contents:
+      /etc/modules-load.d/modules.conf:
+      /etc/sysctl.d/99-sysctl.conf:
+      /etc/systemd/journald.conf:
+      /etc/systemd/logind.conf:
+      /etc/systemd/networkd.conf:
+      /etc/systemd/pstore.conf:
+      /etc/systemd/sleep.conf:
+      /etc/systemd/system-generators/systemd-gpt-auto-generator:
+      /etc/systemd/system.conf:
+      /etc/systemd/user.conf:
+      /etc/xdg/systemd/user:
+      /usr/lib/environment.d/99-environment.conf:
+      /usr/lib/kernel/install.conf:
+      /usr/lib/kernel/install.d/50-depmod.install:
+      /usr/lib/kernel/install.d/55-initrd.install:
+      /usr/lib/kernel/install.d/90-loaderentry.install:
+      /usr/lib/kernel/install.d/90-uki-copy.install:
+      /usr/lib/lsb/init-functions.d/40-systemd:
+      /usr/lib/modprobe.d/systemd.conf:
```

Comment:
This seems specific for `modprobe`: [ref](https://man7.org/linux/man-pages/man5/modprobe.d.5.html). Different slice?